cmd/pmm-dump/main.go

```diff
@@ -123,7 +125,20 @@ func main() {
 		grafanaC := grafana.NewClient(httpC)
 
 		parseURL(pmmURL, pmmHost, pmmPort, pmmUser, pmmPassword)
-		auth(pmmURL, pmmUser, pmmPassword, &grafanaC)
+
+		switch {
+		case *pmmToken != "":
+			log.Info().Msg("Using token authentication")
+			grafanaC.SetToken(*pmmToken)
+		case *pmmCookie != "":
```

Comment:
I wonder if we should allow the user to set both `token` and `cookie`? It's not clear to them which one will win.

Reply:
password > cookie > token

If user provided multiple credentials, does it matter which one was used to login? Hm, probably you're right, it makes sense to accept only one at a time.

Will fix

Reply:
Done